src/Packetery/Module/Checkout.php

```diff
@@ -1413,8 +1413,8 @@ public function filterPaymentGateways( array $availableGateways ): array {
 		}
 
 		$order = null;
-		if ( isset( $wp->query_vars['order-pay'] ) && is_numeric( $wp->query_vars['order-pay'] ) ) {
-			$order = $this->orderRepository->getById( (int) $wp->query_vars['order-pay'], true );
+		if ( isset( $wp->query_vars['order-pay'] ) && is_numeric( $wp->query_vars['order-pay'] ) ) { // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
+			$order = $this->orderRepository->getById( (int) $wp->query_vars['order-pay'], true ); // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```

Comment:
proč tady to ignorujeme? Takhle to nikdo nikdy neopraví. nejde to přesunou t do adapteru?

Reply:
updated